README.md

```diff
@@ -1,189 +1,95 @@
-agent](https://github.com/signalfx/collectd), but still uses that internally on Linux --
-so any existing Python or C-based collectd plugins will still work without
-modification.  On Windows collectd is not included, but the agent is capable of
-running python based collectd plugins without collectd.  C-based collectd plugins
-are not available on Windows.
 
- - [Concepts](#concepts)
  - [Installation](#installation)
- - [Configuration](#configuration)
- - [Logging](#logging)
- - [Proxy Support](#proxy-support)
- - [Diagnostics](#diagnostics)
- - [Development](#development)
-
-## Concepts
-
-The agent has three main components:
-
-1) _Observers_ that discover applications and services running on the host
-2) _Monitors_ that collect metrics from the host and applications
-3) The _Writer_ that sends the metrics collected by monitors to SignalFx.
-
-### Observers
-
-Observers watch the various environments that we support to discover running
-services and automatically configure the agent to send metrics for those
-services.
-
-For a list of supported observers and their configurations,
-see [Observer Config](./docs/observer-config.md).
-
-### Monitors
-
-Monitors collect metrics from the host system and services.  They are
-configured under the `monitors` list in the agent config.  For
-application-specific monitors, you can define discovery rules in your monitor
-configuration. A separate monitor instance is created for each discovered
-instance of applications that match a discovery rule. See [Auto
-Discovery](./docs/auto-discovery.md) for more information.
-
-Many of the monitors are built around [collectd](https://collectd.org), an open
-source third-party monitor, and use it to collect metrics. Some other monitors
-do not use collectd. However, either type is configured in the same way.
-
-For a list of supported monitors and their configurations, 
-see [Monitor Config](./docs/monitor-config.md).
-
-The agent is primarily intended to monitor services/applications running on the
-same host as the agent.  This is in keeping with the collectd model.  The main
-issue with monitoring services on other hosts is that the `host` dimension that
-collectd sets on all metrics will currently get set to the hostname of the
-machine that the agent is running on.  This allows everything to have a
-consistent `host` dimension so that metrics can be matched to a specific
-machine during metric analysis.
-
-### Writer
-The writer collects metrics emitted by the configured monitors and sends them
-to SignalFx on a regular basis.  There are a few things that can be
-[configured](./docs/config-schema.md#writer) in the writer, but this is generally
-only necessary if you have a very large number of metrics flowing through a
-single agent.
+ - [Confirmation](#confirmation)
+ - [Troubleshooting](#troubleshooting-the-installation)
+ - [Next Steps](#next-steps)
+ - [Other methods of Installation](#other-methods-of-installation)
+ 
 
 ## Installation
 
```

Comment:
I think this section needs a bit of intro, along the lines of - The first time you install the SignalFx Smart Agent, we recommend you install on a single host following the instructions in this section. However, if you prefer, you can use a _Linux tar file_ (link) or _Windows zip file_ (link).